readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://NadiaBatrynchuk.github.io/layout_stars/)
+- [TEST REPORT LINK](https://NadiaBatrynchuk.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="937" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/f5d1e8d0-33e8-4f21-b714-7b61b5282961">